src/infra/dex/balancer/dto.rs

```diff
@@ -48,7 +44,6 @@ impl Query {
                 order::Side::Sell => OrderKind::Sell,
             },
             amount: order.amount().amount,
-            gas_price: gas_price.0 .0,
```

Comment:
Seems like this field is required in the current API version: https://github.com/balancer/balancer-api/blob/master/src/modules/sor/types.ts#L9
I tried to send `curl` and it failed without this field.

Fortunately, in API V3 this field is dropped.

Reply:
Thanks for checking. 👍 
Then I'll revert the change for balancer and keep the other two. Since balancer doesn't send us huge bills this is not so important anyway. 😅